### PR TITLE
Comment out graupel warning

### LINKED ISSUE #866: Graupel precipitation warning print statement
When the warning "Graupel is higher than total precip" is being printed, it is printed too often which slows down RRFS. This warning is being commented out. I think that this warning message prints only when graupel alone is reaching the surface. Total precipitation is interpolated and precipitation from individual hydrometeor categories is not. Thus, when total precipitation equals graupel precipitation and total precipitation is interpolated and graupel precipitation is not, the two values may not be equal. Additionally, the values of graupel precipitation were changed in the print statement to graup_bucket(i,j)*1.e-3, to have the correct units.

### PR DESCRIPTION
**Pull request by AndersJensen-NOAA**
This PR fixes issue https://github.com/NOAA-EMC/UPP/issues/866

### CODE REVIEW

**Comment by FernandoAndrade-NOAA** [comment]
Hi @AndersJensen-NOAA, please link/create a git issue and add a description to the PR description itself. Changes to /sorc files will also need a changelog addition in the beginning section before approval, thank you! This helps with managing and tracking of updates.

**Review by tanyasmirnova — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
@AndersJensen-NOAA Per UPP code contribution procedure ([here](https://github.com/NOAA-EMC/UPP/wiki/UPP-Code-Development#contribute-changes-to-other-upp-branches)), please open an UPP issue with a brief description of the issue you are addressing at https://github.com/NOAA-EMC/UPP/issues. Thanks!

File: sorc/ncep_post.fd/SURFCE.f
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/SURFCE.f:1]: @AndersJensen-NOAA Please add a change log entry with your name after line 48 in this file.

**Comment by WenMeng-NOAA** [comment]
The UPP RTs were completed on WCOSS2 without changed results.

**Comment by AndersJensen-NOAA** [comment]
This PR fixes issue #866

**Comment by WenMeng-NOAA** [comment]
@FernandoAndrade-NOAA You may start the UPP RTs on Hera and Orion. There should be no baseline update with this PR.

**Comment by FernandoAndrade-NOAA** [comment]
> @FernandoAndrade-NOAA You may start the UPP RTs on Hera and Orion. There should be no baseline update with this PR.

Hera and Orion UPP RTs completed without changes to results. Looks good on my side!

**Review by FernandoAndrade-NOAA — APPROVED** [review]

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
This PR is ready for merging.

**Pull request merged**